firebase-storage/src/test/java/com/google/firebase/storage/TestUtil.java

```diff
@@ -148,6 +154,6 @@ static void await(Task<?> task, int timeout, TimeUnit timeUnit) throws Interrupt
    * Tasks to be executed.
    */
   static void await(Task<?> task) throws InterruptedException {
-    await(task, 3, TimeUnit.SECONDS);
+    await(task, 10, TimeUnit.SECONDS);
```

Comment:
why? 

don't forget to update the javadoc

Reply:
This is to compensate for the backoff we just implemented. Updated the doc